tests/test_routes.py

```diff
@@ -235,3 +242,80 @@ def test_route_sub_criteria_side_bar(
         assert (
             expected_names in response.data
         ), "Response does not contain expected name"
+
+    def test_route_sub_criteria_scoring_inaccessible_to_commenters(
+        self, flask_test_client
+    ):
+
+        # Mocking fsd-user-token cookie
+        test_payload = {
+            "accountId": "test-user",
+            "email": "test@example.com",
+            "fullName": "Test User",
+            "roles": ["COMMENTER"],
+        }
```

Comment:
Can we use the `test_commenter_claims` constant so we're not repeating this? 

https://github.com/communitiesuk/funding-service-design-assessment/blob/037a5d69c15a808a3e9a23186c0ebc73b49e2264/tests/conftest.py#L31-L36